src/oas.md

```diff
@@ -44,6 +44,27 @@ Each template expression in the path MUST correspond to a path parameter that is
 
 The value for these path parameters MUST NOT contain any unescaped "generic syntax" characters described by [RFC3986](https://tools.ietf.org/html/rfc3986#section-3): forward slashes (`/`), question marks (`?`), or hashes (`#`).
 
+The path templating expression is defined by the following [ABNF](https://tools.ietf.org/html/rfc5234) syntax
```

Comment:
```suggestion
The path templating is defined by the following [ABNF](https://tools.ietf.org/html/rfc5234) syntax
```

Isn't it either `path templating` or `path template expression`?  Mixing it together doesn't sound right. This ABNF is defining whole `path templating`, not just the `path templating expression`.

My suggestion is aligned with https://github.com/OAI/OpenAPI-Specification/blob/1da9fbbbce16ea22da73951fdefdd1d32c447664/src/oas.md#runtime-expressions